### PR TITLE
[DO-1861]  [Contains migration] Improve pep model

### PR DESCRIPTION
**Pull request by Iva-khar**
- field `pep.sanctions` without 'null' value, with '' instead
- migration for convertind 'null' to ''
- pep converter with '' if sanctions is absent

### CODE REVIEW

File: business_register/converter/pep.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def save_to_db(self, json_file):
             if last_employer:
                 last_employer = last_employer.lower()
             info = pep_dict.get('wiki_uk')
-            sanctions = pep_dict.get('reputation_sanctions_uk')
+            sanctions = pep_dict.get('reputation_sanctions_uk') or ''
```
Comment (EmmaSamoilova) [inline business_register/converter/pep.py:107]: maybe sanctions = pep_dict.get('reputation_sanctions_uk', '') ?

Comment (Iva-khar) [reply business_register/converter/pep.py]: yes. done.

**Review by EmmaSamoilova — APPROVED** [review]

**Comment by roman-tiukh** [comment]
same comment as in #1155

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
same comment as in #1155

**Comment by Iva-khar** [comment]
Add method for change null value to '' in migration

File: business_register/migrations/0155_auto_20210802_1022.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        model = apps.get_model('business_register', 'Pep')
+        for instance in model.objects.all():
+            if not getattr(instance, 'sanctions'):
+                setattr(instance, 'sanctions', '')
+                instance.save(update_fields=['sanctions'])
+        historical_model = apps.get_model('business_register', 'HistoricalPep')
+        for historical_instance in historical_model.objects.all():
+            if not getattr(historical_instance, 'sanctions'):
+                setattr(historical_instance, 'sanctions', '')
+                historical_instance.save(update_fields=['sanctions'])
```
Comment (roman-tiukh) [inline business_register/migrations/0155_auto_20210802_1022.py:22]: move this function from Migration class to module

Comment (roman-tiukh) [reply business_register/migrations/0155_auto_20210802_1022.py:22]: https://github.com/3v-workspace/Data_converter/pull/1155/files#r682336319

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/migrations/0155_auto_20210802_1022.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/migrations/0155_auto_20210802_1022.py]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**